robot/web/stream/state/state.go

```diff
@@ -409,3 +409,8 @@ func (state *StreamState) unsubscribeH264Passthrough(ctx context.Context, id rtp
 
 	return nil
 }
+
+// IsResized returns whether the stream is in a resized state.
+func (state *StreamState) IsResized() bool {
+	return state.isResized
+}
```

Comment:
Is this thread safe?

Reply:
I think we are thread safe here. isResized is only modified by the `sourceEventHandler` routine and we are not locking or anything. I think there could be some super edge case where a Reset or SetStreamOptions hits at the same time a reconfigure happens causing this look up to be inaccurate.